### PR TITLE
WIP Add admin style guide

### PR DESCRIPTION
**Pull request by Sinetheta**
screen shot 2015-12-01 at 10 50 16 am

Finishes #533

Waiting on https://github.com/solidusio/solidus/pull/509

### CODE REVIEW

**Comment by Mandily** [comment]
It would be nice if we had the option to make the code area full width or half width. I think there may be some cases where images are smaller and there's a lot of code. It would be nice to not have to scroll to see it.

File: backend/vendor/assets/stylesheets/prism.css [hunk 1]
```diff
@@ -0,0 +1,139 @@
+/* http://prismjs.com/download.html?themes=prism&languages=markup+css+clike+javascript+ruby */
```
Comment (jhawthorn) [inline backend/vendor/assets/stylesheets/prism.css:1]: I am a cranky butt and don't want syntax hilighting

**Comment by Sinetheta** [comment]
Waiting on https://github.com/solidusio/solidus/pull/509

**Comment by Sinetheta** [comment]
Closed for legit PR https://github.com/solidusio/solidus/pull/602

**Pull request closed without merging**